packages/app/tasks/get-pool-pending-rewards.js

```diff
@@ -0,0 +1,318 @@
+const fs = require('fs');
+const BN = require('bn.js');
+const { set, unset } = require('lodash');
+
+const { trace, info, error, warning, arg } = require('../utils/logger');
+const DB = require('../utils/db');
+
+const BATCH_SIZE = 500;
+
+const getPoolPendingRewardsTask = async (env, { poolToken }) => {
+    const eq = (address1, address2) => {
+        return address1.toLowerCase() === address2.toLowerCase();
```

Comment:
IMO, there's no need to use `toLowerCase` if we make sure to write properly every hard-coded address in our code, and to use `web3.utils.toChecksumAddress` on every non-hard-coded address that we generate during runtime (for example, from event info).

Reply:
This is run against external JSON files and a very sensitive operation, so I want to avoid any error-prone assumptions.